zip-0252.rst

```diff
@@ -142,28 +130,34 @@ NU5 activation. Both transaction formats MUST be accepted by NU5 nodes.
 Backward compatibility of the new ``MSG_WTX`` inv type introduced for ``inv``
 and ``getdata`` messages is discussed in [#zip-0239]_.
 
-Backward compatibility of the new ``addrv2`` message (not technically part of
-NU5) is discussed in [#zip-0155]_.
-
 
 Support in zcashd
 =================
 
-Support for NU5 on Testnet is implemented in `zcashd` version 4.5.1, which
-advertises protocol version 170015. (A previous revision of the NU5 consensus
-rules was implemented in `zcashd` version 4.5.0, but this revision contained
-critical bugs in the Orchard Action circuit. Before it could activate, `zcashd`
-version 4.5.1 was released, with the later NU5 Testnet activation height and
-consensus branch ID defined in this document.)
+Several versions of `zcashd` have implemented versions of the NU5 consensus
+rules on Testnet:
+
+- `zcashd` v4.5.0 implemented a consensus revision that contained critical
+  bugs in the Orchard Action circuit.
+- Before that revision could activate, `zcashd` v4.5.1 was released, with a
+  later activation height of 1599200 as described in section `NU5 deployment`_
+  above. This revision also had a consensus bug that caused many nodes to
+  stall at or near height 1779199, shortly after the first block containing
+  an Orchard output.
+- `zcashd` v4.7.0 will implement what is expected to be the final revision
+  of the NU5 consensus rules, causing a long rollback to an alternate
+  Testnet chain. It will be necessary to use the ``-reindex`` option to
+  `zcashd` in order to follow this chain as intended.
 
 Support for NU5 on Mainnet will be implemented in `zcashd` version 5.0.0,
-which will advertise protocol version 170017.
+which will advertise protocol version 170100.
 
 Backward compatibility in zcashd
 --------------------------------
 
 The minimum peer protocol version that NU5-compatible `zcashd` nodes will
-connect to is 170002.
+connect to is 170002. They will immediately disconnect from Testnet nodes
+advertising a peer protocol version less than 170040.
```

Comment:
Zebra will also have a minimum of 170,040:
https://github.com/ZcashFoundation/zebra/pull/3976

Would you mind updating the Zebra section below?